package.json

```diff
@@ -15,38 +15,52 @@
   "dependencies": {
     "@popperjs/core": "2.11.5",
     "classnames": "2.3.1",
-    "immutability-helper": "3.1.1",
-    "jquery": "3.6.0",
     "js-cookie": "3.0.1",
-    "leaflet": "1.7.1",
-    "leaflet-draw": "1.0.4",
-    "leaflet.markercluster": "1.5.3",
-    "prop-types": "15.8.1",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
-    "react-markdown": "8.0.3",
-    "react-popper": "2.2.5",
-    "react-slick": "0.28.1",
-    "shpjs": "4.0.2",
-    "slick-carousel": "git+https://github.com/liqd/slick.git#pm-2019-07-overwrites"
+    "prop-types": "15.8.1"
   },
   "devDependencies": {
     "@babel/plugin-transform-runtime": "^7.17.0",
     "@babel/preset-env": "7.17.12",
     "@babel/preset-react": "7.17.12",
     "@testing-library/react": "12.1.5",
     "babel-preset-react": "6.24.1",
-    "eslint": "8.15.0",
+    "eslint": "7.32.0",
     "eslint-config-standard": "16.0.3",
     "eslint-config-standard-jsx": "10.0.0",
     "eslint-plugin-import": "2.26.0",
     "eslint-plugin-node": "11.1.0",
-    "eslint-plugin-promise": "6.0.0",
+    "eslint-plugin-promise": "5.2.0",
     "eslint-plugin-react": "7.29.4",
     "eslint-plugin-standard": "4.1.0",
     "husky": "7.0.4",
+    "immutability-helper": "3.1.1",
     "jest": "27.5.1",
-    "lint-staged": "12.4.1"
+    "leaflet": "1.7.1",
+    "jquery": "3.6.0",
+    "leaflet-draw": "1.0.4",
+    "leaflet.markercluster": "git+https://github.com/liqd/Leaflet.markercluster#5ed89b26922c51083fc9632a2c01425b9261a0f5",
```

Comment:
@phillimorland  we use our fork in a+ but the upstream repo in a4, was there a reason for that? I changed it to the fork as well for now

Reply:
@goapunk only thing I can think of is a+ have specific styling but that should be in this package